docs/release-notes/NuGet-5.4.md

```diff
@@ -23,6 +23,8 @@ NuGet distribution vehicles:
 
 * New helper function - given a list of package ids and versions, get the likely top level packages. - [#8316](https://github.com/NuGet/Home/issues/8316)
 
+* New [`nuget/setup-nuget`](https://github.com/marketplace/actions/setup-nuget-exe-for-use-with-actions) action for installing and configuring NuGet.exe on [GitHub Actions](https://github.com/features/actions).
```

Comment:
add a link to the issue that tracked this work. See other items for how we do this.